core/config-manager/unmarshal.go

```diff
@@ -283,9 +327,109 @@ func (cMgr *ConfigurationManager) populateMap(prefix string, mapType reflect.Typ
 	return rValue, nil
 }
 
-func checkPrefix(heap, prefix string) (bool, int) {
+func isSliceContainString(str string, list []string) bool {
+	for _, value := range list {
+		if value == str {
+			return true
+		}
+	}
+	return false
+}
+
+func checkAndReplaceInline(prefix, heap string, tagList []string) (string, string) {
```

Comment:
plz use go cyclo to chech this file

Reply:
Verified go cyclo.
No issues found for this file.